hacks/better_dbref.js

```diff
@@ -0,0 +1,25 @@
+DBRef.prototype.__toString = DBRef.prototype.toString;
+DBRef.prototype.toString = function () {
+  var org = this.__toString();
+  if ( ! mongo_hacker_config.better_dbref.on) {
+    return org;
+  }
+  var additional = [];
```

Comment:
consider pulling mongo_hacker.better_dbref into a var up here to make the subsequent uses of db_if_differs, plain, etc easier to read